src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -1254,7 +1254,7 @@ def write_sorting(
 
 
 def add_waveforms(
-    waveform_extractor: WaveformExtractor,
+    waveform_extractor,
```

Comment:
```suggestion
    waveform_extractor: "WaveformExtractor",
```
Should we at least do this?

Reply:
Yeah, I think that's better. Let me push this.

Reply:
Or else nest entirely in a try/except scope

try:
    from spikeinterface import WaveformExtractor

    def add_waveforms(waveform_extractor: WaveformExtractor, ...)
    ...
except Exception as exception:
    assert type(exception) is <type of exception - I think an ImportError?)
    assert "Some string part of the message unique to the failured import of waveforms extractor" in str(exception)

Reply:
Or, or,

LIke you do in tests, nest entirely in `if SI_VERSION > threshold` block?

Reply:
I rather just add the docstring. I don't think is worth it with an object that will be dropped anyway.